app/assets/javascripts/shared/datatable/state.js

```diff
@@ -1,59 +1,84 @@
 $(document).on('preInit.dt', 'body.issues.index', function (e, settings) {
   var $paths = $('[data-behavior~=datatable-paths]');
-  if ($paths.data('table-state-url') == undefined) { return; }
+  if ($paths.data('table-state-url') == undefined) {
+    return;
+  }
+
+  var api = new $.fn.dataTable.Api(settings),
+    isQA = $('[data-behavior~=qa-viewer]').length > 0;
 
-  var api = new $.fn.dataTable.Api( settings );
   api.button().add(1, {
     attr: {
-      'data-behavior': 'table-action'
+      'data-behavior': 'table-action',
```

Comment:
Are [trailing commas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) the convention?

Reply:
It seems like it became the default when [ES5](https://www.w3schools.com/js/js_es5.asp) was released.